exo*: add pages (#16405)
tldr-pages/tldr: pages/common/exo-compute-sks.md

# exo compute sks

> > CLI for managing the Exoscale Scalable Kubernetes Service (SKS).

Reviewer: ```suggestion
> Manage the Exoscale Scalable Kubernetes Service (SKS).
```

> > More information: <https://community.exoscale.com/community/sks/>.

Reviewer 2: ```suggestion
> More information: <https://community.exoscale.com/reference/cli/exo/compute/sks>.
```
404

- List the supported SKS cluster versions:

`exo compute sks versions`

- Create a new SKS cluster:

`exo compute sks create {{cluster_name}} {{[-z|--zone]}} {{zone}}`

- List all SKS clusters:

`exo compute sks list`

> - Generate a Kubernetes kubeconfig file for an SKS cluster that expires in 30 minutes:

Reviewer 3: ```suggestion
- Generate a Kubernetes kubeconfig file for an SKS cluster that expires in 1800 seconds:
```
